### DISCUSSION TITLE
Firefly III trying to display data from year 0001

### DISCUSSION THREAD

**Discussion by bedlamzd** [firefly-iii/firefly-iii#6288] [Firefly III] [answered]
After update to 5.7.10 firefly III started trying to display data from year 0001 even though transactions start in 2017.

reports page, for example

![image](https://user-images.githubusercontent.com/44726005/182847779-fe88f34e-a660-4382-9811-7952d6385c31.png)

Not sure what exactly to look for, logs seem ok

**Comment by JC5** [comment, marked as answer]
You have a transaction somewhere from that year, or earlier. Check out the transaction lists. You'll find it at the bottom of the list.

**Reply by bedlamzd** [reply]
Wow, I thought I've looked good (I even opened mariadb transactions table and found nothing)

thank you
